engine/exec/exec.go

```diff
@@ -21,21 +21,9 @@ func Run(ctx context.Context, step *spec.Step, output io.Writer, isDrone bool) (
 		return nil, errors.New("step entrypoint cannot be empty")
 	}
 
-	// used for printing out the command
-	command := make([]string, len(step.Command))
-	copy(command, step.Command)
-
 	// we shouldn't change this functionality for drone.
```

Comment:
i would change the comment here, to what was previously on line 24, 
eg `#for CIE we printout the command we are executing`